SPECS/ocaml-cinaps/ocaml-cinaps.spec

```diff
@@ -1,27 +1,28 @@
 %global srcname cinaps
 
-# TESTING NOTE: The ppx_jane module is needed to run the tests.  However,
-# ppx_jane transitively requires this module.  Therefore, we cannot run the
-# tests at all until we are able to add ppx_jane.
-%ifnarch %{ocaml_native_compiler}
-%global debug_package %{nil}
-%endif
+# This package is needed to build ppx_jane, but its tests require ppx_jane.
+# Break the dependency cycle here.
+%bcond_with test
```

Comment:
Same case - test-only dependencies shouldn't cause cyclic dependencies when behind `%{with_check}`.

Reply:
Removed it